pulp_service/pulp_service/app/models.py

```diff
@@ -174,3 +174,17 @@ class VulnerabilityReport(models.Model):
 
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     vulns = models.JSONField()
+    
+class AnsibleLogReport(models.Model):
```

Comment:
@YasenT We have a BaseModel that can be inherited here and avoid some duplication. Check it https://github.com/pulp/pulpcore/blob/18adef4f5c89d2c6c746fa635b50bb324259ee20/pulpcore/app/models/base.py#L20